ui_framework/components/modal/_modal.scss

```diff
@@ -4,6 +4,7 @@
   border: 1px solid $modalBorderColor;
   border-radius: $globalBorderRadius;
   box-shadow: 0 5px 22px rgba(#000000, 0.25);
+  z-index: 1001;
```

Comment:
Can you create a variable called `$globalModalDepth` in the variables file and assign this value there?

@snide Just thought you'd like to see me making this kind of snideresque comment. Cherish this moment!